style.css

```diff
@@ -1,29 +1,44 @@
-.hljs.line-numbers,
-.hljs.selected-lines {
+.wp-block-code > div {
+	overflow: auto;
+}
+
+.hljs.shcb-code-table {
 	display: table;
 	width: 100%;
```

Comment:
@allejo Is the `width:100%` required? I'm finding in some themes, like Twenty Fourteen, it causes a bit of horizontal scrolling even when line wrapping is enabled. Can we remove it?

```suggestion
```

Reply:
Humm. Seems like it is needed. If line numbers are turned off and the contents don't cause any wrapping, then the table is not wide enough:

![image](https://user-images.githubusercontent.com/134745/80312503-46c12d80-879a-11ea-85e6-159e34cd2030.png)

Reply:
I found the problem with Twenty Fourteen: the `.hljs` element wasn't getting `box-sizing:border-box`, so the combination of `width:100%` and the `padding` resulted in an `overflow`. Fixed in e236851.